app/src/test/java/com/twilio/video/app/participant/ParticipantManagerTest.kt

```diff
@@ -0,0 +1,106 @@
+package com.twilio.video.app.participant
+
+import org.hamcrest.CoreMatchers.equalTo
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Assert.fail
+import org.junit.Test
+
+class ParticipantManagerTest {
```

Comment:
Excellent 👍

Reply:
Thanks! So glad we can finally test some of this logic!